dead_apps/clock/index.html

```diff
@@ -15,48 +15,36 @@
     <link rel="stylesheet" type="text/css" href="style/picker/picker.css">
 
     <!-- Shared styles -->
-    <link rel="stylesheet" type="text/css" href="shared/style/status.css">
-    <link rel="stylesheet" type="text/css" href="shared/style/tabs.css">
+    <link rel="stylesheet" type="text/css" href="style/status.css">
```

Comment:
If we are using the ../shared path in the other files, lets do it in this one too